### DISCUSSION TITLE
Urgent Help! We are trying to store the data from grapeJS editor in Angular application, getting CORS error, image attached below

### DISCUSSION THREAD

**Discussion by mastercorp-usa** [GrapesJS/grapesjs#4190] [Q&A]
source code we used:
   
 var editor = grapesjs.init({
      container: '#gjs',
      style: '.txt-red{color: red}',
      plugins: ['gjs-preset-newsletter', 'gjs-blocks-basic'],
      pluginsOpts: {
        'gjs-blocks-basic': { blocks: ['video'] }
      },
      assetManager: {
        assets: [
          'https://siteurl.edu/uploads/screen-shot-2017-11-29-at-9-17-08-am.png',
          'https://siteurl.edu/uploads/screen-shot-2017-11-16-at-9-21-05-am.png',
        ],
        storeOnChange: true,
        storeAfterUpload: true,
        upload: "/upload-editor-media",
      },
      storageManager: {
        type: 'remote',
        stepsBeforeSave: 2,
        storeComponents: false,
        storeStyles: true,
        storeHtml: true,
        storeCss: true,
        contentTypeJson: true,
        headers: {
          Authorization: `Bearer ${this.currentUser.accessToken}`
        },
        Pragma: 'no-cache',
        Expires: '0',
        urlStore: "https://a8devapi.azurewebsites.net/api/EC_Email_Templates/GrapeJS",
        urlLoad: "/load-template/2",
        params: {},   // For custom values on requests

      }
    });

by using this code getting error CORS, like below

![Screenshot (54)](https://user-images.githubusercontent.com/74088643/157726688-7d1c7faa-4767-43f0-8650-d8e8c68bf438.png)

**Comment by ronaldohoch** [comment]
CORS issue are solved in server.
Your server should add the editor in a whitelist on CORS config.
